docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,61 @@
+[//]: # (title: Troubleshooting)
+
+Most of the browsers support WebAssembly. For recent browser versions, WebAssembly works by default. However,
+for older browser versions, you still need to update the browser settings.
+
+## Browser versions
+
+To run a Kotlin/Wasm project, check if WebAssembly works with the browser version by default. Otherwise, update the 
```

Comment:
The point is that Kotlin/Wasm depends on new proposals (Wasm GC, Exception handling, Typed Function Reference), and it may require updating or setting up an environment.

Reply:
Also rephrased here. WDYT?